src/StyledFormComponents.js

```diff
@@ -32,4 +42,8 @@ const StyledCard = styled.div`
   position: relative;
 `;
 
-export { StyledCardHeading, StyledForm, StyledCard };
+const Card = ({ children, ...props }) => {
```

Comment:
I don't think this component is needed, wherever you call this card I think you can write it as `<StyledCard {...props}>{ all your children needed } </StyledCard>` as it is already a component